packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -165,7 +165,14 @@ class OnsetPicker extends React.Component {
    * @param {precision} - New precision value
    */
   changePrecision(precision) {
-    this.setState({ precision }, this.handleOnsetUpdate);
+    if (precision !== PrecisionOptions.UNKNOWN) {
```

Comment:
This is up to your preference, but when possible I think taking the truthy path is usually easier to understand. I'd recommend flipping this statement so we always perform an action when it is Unknown rather than if it's not.

```jsx
if (precision === PrecisionOptions.UNKNOWN) {
  // Logic for Unknown
} else {
  // Logic for any other precision
}
```

Reply:
Resolved in 6c98210